src/QMCTools/QMCGaussianParserBase.cpp

```diff
@@ -1925,11 +1939,11 @@ void QMCGaussianParserBase::dumpStdInputProd(const std::string& psi_tag, const s
         xmlAddChild(initopt, parameter(initopt, "warmupSteps", "2"));
         xmlAddChild(initopt, parameter(initopt, "timestep", "0.5"));
         xmlAddChild(initopt, parameter(initopt, "walkers", "1"));
-        xmlAddChild(initopt, parameter(initopt, "samples", "8000"));
+        xmlAddChild(initopt, parameter(initopt, "samples", "80000"));
```

Comment:
Do we really need to changes these parameters in the generated input files?

Reply:
yes... Unfortunately I find myself changing those all the time! In fact there are 2 type of users, the ones who use the files generated by convert4qmc (that would be only me for molecules) and the ones using nexus (the majority of people including me for solids) 

Nexus does not care about those. And for me, I use 80000 samples all the time as 8000 often is not enough.

Reply:
No fixed sample count can apply to all systems, even all "small" ones, so I view this as further indication that "we are doing it wrong".  Until there are simpler inputs that specify target energy errors etc we will continue to have this problem.

I don't might what we merge here.

Reply:
Keep in mind that this is only generated with the flag "-production" as a way to provide a very "conservative" starting point. but I agree that having less parameters to control and a dynamic sample population that can increase or decrease with the complexity of the problem would be great.  the choice of "increasing" the samples is based solely on the type of calculations I have been running lately and obviously will not work for all class of problems

Reply:
I don't mind this change. My overall thinking about such cases are
If some changes don't matter much, not changing is better than changing because any change has related questions like why? impact?

Reply:
ah.. I understand now. It is just  a bit annoying for me to always change the sample size but it is not the end of the world. This particular change is really not important to me, I added it since there was a bunch of small changes.. if you want to revert it it is fine with me.